docs/bbob-largescale/functions/source/index.rst

```diff
@@ -375,16 +380,16 @@ dimensions 20, 40, 80, 160, 320, and 640 of the ``bbob-largescale`` test suite.
 
 Implementation
 --------------
-Now, we describe how to implement the replacement of the rotational transformations in the
-``bbob`` test suite with the realizations of :math:`P_{\text{left}}BP_{\text{right}}`. This will be illustrated through an example
-on the Ellipsoidal function (rotated) :math:`f_{10}(\mathbf{x})` (see Table in the next section), which is defined by
+Now, we describe how these changes to the rotational transformations are implemented.
```

Comment:
Now the text is less clear than before to me. Is it clear what "these changes" are?

Reply:
Now slightly modified.